HtmlBuilderCore/invalidInputException.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HtmlBuilderCore
+{
+    public class invalidInputException : Exception
```

Comment:
Good. You met the MSDN guideline of putting a file it it's own class. Ensure again the casing is right. It should be `InvalidInputException`